### DISCUSSION TITLE
Why can't gif be displayed on the right by default?

### DISCUSSION THREAD

**Discussion by imingx** [community/community#23477] [New to GitHub]
<p><a href="https://github.blog/changelog/2022-05-19-option-to-prevent-animated-images-from-playing-automatically/" rel="noopener nofollow ugc">Option to prevent animated images from playing automatically</a></p>
<p>Because of this update↑, the following options have been created.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/4/d4b14ae3762acd6a63f36c54c61cf76a1c35d2f0.png" data-download-href="/uploads/short-url/ulz4pPw3W5YrJx66zjvAm7uhtv2.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/54981719/181077294-0c69b016-49f2-406d-84e1-b71d58f772dc.png" alt="image" data-base62-sha1="ulz4pPw3W5YrJx66zjvAm7uhtv2" width="313" height="250" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/4/d4b14ae3762acd6a63f36c54c61cf76a1c35d2f0_2_313x250.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/4/d4b14ae3762acd6a63f36c54c61cf76a1c35d2f0_2_469x375.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/4/d4b14ae3762acd6a63f36c54c61cf76a1c35d2f0_2_626x500.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/4/d4b14ae3762acd6a63f36c54c61cf76a1c35d2f0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">822×656 43.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>When the user is not logged in or not set, the default is <code>Sync with system</code>. But this will cause all gifs to become left-aligned. like:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/0/f08c39a03d8e14c692c3d942a9f5f769690f54f4.png" data-download-href="/uploads/short-url/yjZ1tuDeRqSBmNNRwQD1KNMAtfu.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/54981719/181077295-03055e73-6de6-4989-94c4-e74e4c49211c.png" alt="image" data-base62-sha1="yjZ1tuDeRqSBmNNRwQD1KNMAtfu" width="690" height="216" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/0/f08c39a03d8e14c692c3d942a9f5f769690f54f4_2_690x216.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/0/f08c39a03d8e14c692c3d942a9f5f769690f54f4_2_1035x324.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/0/f08c39a03d8e14c692c3d942a9f5f769690f54f4_2_1380x432.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/0/f08c39a03d8e14c692c3d942a9f5f769690f54f4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1798×564 76.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>But actually, the gif is already set to right-aligned.</p>
<pre><code class="lang-bash">&lt;img src="*.gif" align="right"&gt;
</code></pre>
<p>when I set the options to <code>Enabled</code>, it displays properly on the right side:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/c/dc4dc84be9da20f5fe568321adbc1a63302623e5.png" data-download-href="/uploads/short-url/vqTGShmWgpzTKLIijvAsfbEcl0h.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/54981719/181077296-79dfe5ce-4854-45fd-a2b6-aaf3060082ac.png" alt="image" data-base62-sha1="vqTGShmWgpzTKLIijvAsfbEcl0h" width="690" height="210" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dc4dc84be9da20f5fe568321adbc1a63302623e5_2_690x210.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dc4dc84be9da20f5fe568321adbc1a63302623e5_2_1035x315.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dc4dc84be9da20f5fe568321adbc1a63302623e5_2_1380x420.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dc4dc84be9da20f5fe568321adbc1a63302623e5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1816×554 87.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hello - and thank you for this feedback!</p>
<p>Please submit your feedback via the new GitHub <a href="https://github.com/github/feedback/discussions">Feedback Discussions forum</a> so that our product team can better track your request :slight_smile:. It’s a new thing, but we’re trying to push all product feedback to the same new area so it’s easier to find and consolidate user feedback!</p>

**Comment by paveloom** [comment]
<p>Just hit this issue, too. Here’s what I have in my <a href="https://github.com/Paveloom/Paveloom" rel="noopener nofollow ugc"><code>README.md</code></a>:</p>
<pre><code class="lang-auto">&lt;div align="center"&gt;
  &lt;a rel="nofollow" href="https://paveloom.github.io/git/"&gt;
    &lt;img width="50%" src="./levitate.gif"&gt;
  &lt;/a&gt;
&lt;/div&gt;
</code></pre>
<p>And here’s the difference between the settings.</p>
<p>Motion &gt; Autoplay animated images &gt; Sync with system:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/4/743f68a05478394dbbb5ce56bf09a088adb27406.png" data-download-href="/uploads/short-url/gAneP0TYTuHnWLNX4N1KkCnc8rc.png?dl=1" title="sync" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/49961859/181077298-13673a5c-274d-4dfa-899d-77b53b4dbfc3.png" alt="sync" data-base62-sha1="gAneP0TYTuHnWLNX4N1KkCnc8rc" width="690" height="318" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/4/743f68a05478394dbbb5ce56bf09a088adb27406_2_690x318.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/4/743f68a05478394dbbb5ce56bf09a088adb27406_2_1035x477.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/4/743f68a05478394dbbb5ce56bf09a088adb27406.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/4/743f68a05478394dbbb5ce56bf09a088adb27406_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">sync</span><span class="informations">1210×559 40.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Motion &gt; Autoplay animated images &gt; Enabled:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/8/28b2460d9657c2068a91db1d2fa71bc58c1e3250.png" data-download-href="/uploads/short-url/5O12ZOT02KspEkWY0eYYIjePVLy.png?dl=1" title="enabled" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/49961859/181077300-eeee74fd-64c4-42a5-90ef-fd45431d508f.png" alt="enabled" data-base62-sha1="5O12ZOT02KspEkWY0eYYIjePVLy" width="690" height="318" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/8/28b2460d9657c2068a91db1d2fa71bc58c1e3250_2_690x318.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/8/28b2460d9657c2068a91db1d2fa71bc58c1e3250_2_1035x477.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/8/28b2460d9657c2068a91db1d2fa71bc58c1e3250.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/8/28b2460d9657c2068a91db1d2fa71bc58c1e3250_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">enabled</span><span class="informations">1213×560 40.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>The link doesn’t work in the first option either.</p>
